tests/accounts/test_password_reset.py

```diff
@@ -0,0 +1,27 @@
+from django.test import Client
+from django.urls import reverse
+
+import pytest
+
+from maykin_common.accounts.password_reset import PasswordResetView
+
+pytestmark = [pytest.mark.urls("tests.accounts.urls")]
+
+
+def test_user_cant_access_the_password_reset_view_more_than_5_times(client: Client):
+    url = reverse("admin_password_reset")
+    for _ in range(5):
+        response = client.get(url)
+        assert response.status_code == 200
+
+    response = client.get(url)
+    # 429 Too Many Requests
+    assert response.status_code == 429
```

Comment:
```suggestion

    # 429 Too Many Requests
    assert response.status_code == 429
```
visual separation between thing-being-tested and assertions